QC text updates (#568)
AlexsLemonade/scpca-nf: templates/qc_report/celltypes_supplemental_report.rmd

## Unsupervised clustering

> Here we show the labels from unsupervised clustering to against cell type annotations.

Reviewer: ```suggestion
Here we show the labels from unsupervised clustering compared to cell type annotations.
```

We therefore expect reliable labels to have values close to 1.

The plot below shows the distribution of `CellAssign`-calculated probabilities for the final cell type labels.
Line segments represent individual values that comprise each distribution.

> Note that no distrubution is shown for cell types with 2 or fewer labeled cells; only line segments are shown for such distributions.

Reviewer: How did "distrubution" get past our spell checks? (It may have snuck into the dictionary).

Nonetheless, we can maybe make this more compact.
```suggestion
For cell types with 2 or fewer labeled cells, only the individual value line segments are shown.
```

Reviewer: > How did "distrubution" get past our spell checks? (It may have snuck into the dictionary).

oh `knitr::as_is` strikes again

Author: > strikes again

yeah...it's a problem...
